### PR TITLE
mastodon: update ruby package blurhash

### PR DESCRIPTION
**Pull request by Izorkin**
###### Description of changes
Issue - https://github.com/NixOS/nixpkgs/issues/223534

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by chvp — APPROVED** [review]
Added this patch to my config and deployed. Everything works as expected. Thanks!

File: pkgs/servers/mastodon/source.nix [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
     url = "https://github.com/mastodon/mastodon.git";
     rev = "v4.1.1";
     sha256 = "1c0mxz45pkgmyw81z025n1ps1dkdq92337h7sd865w2fbgmzg50l";
   };
 in applyPatches {
   inherit src;
-  patches = [];
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/mastodon/mastodon/commit/61b069b20ef6977703721ae44ae8a68be6bb0992.patch";
```
Comment (erictapen) [inline pkgs/servers/mastodon/source.nix:12]: Could you please use https://github.com/mastodon/mastodon/commit/e7b81d7d9625893b1323e12215a2a98c0f19f58f ?
It's the same commit, but the one merged into `main`, so it's easy to see in which release it got included. Makes removing it later easier.

**Review by erictapen — CHANGES_REQUESTED** [review]

File: pkgs/servers/mastodon/source.nix (hunk 1 above)
Comment (Izorkin) [reply pkgs/servers/mastodon/source.nix]: Fixed.

**Review by marsam — APPROVED** [review]

**Comment by erictapen** [comment]
Thanks!

**Pull request merged**